js/main.js

```diff
@@ -126,3 +130,52 @@ function editEntry(newData, DOMElement) {
   $editTitle.textContent = newData.title;
   $editNotes.textContent = newData.notes;
 }
+
+$delete.addEventListener('click', function (event) {
+  addModal();
+});
+
+function addModal() {
+  var $confirmationModal = $containerElements[0].appendChild(document.createElement('div'));
+  $confirmationModal.setAttribute('class', 'confirmation-modal');
+  var $modal = $confirmationModal.appendChild(document.createElement('div'));
+  $modal.setAttribute('class', 'modal');
+  var $modalText = $modal.appendChild(document.createElement('h3'));
+  $modalText.textContent = 'Are you sure you want to delete this entry?';
+  var $cancelButton = $modal.appendChild(document.createElement('button'));
+  $cancelButton.setAttribute('class', 'cancel-button');
+  $cancelButton.textContent = 'cancel';
+  var $deleteButton = $modal.appendChild(document.createElement('button'));
+  $deleteButton.setAttribute('class', 'delete-button');
```

Comment:
This does not need to be dynamically created, you can write the delete modal directly in your HTML file, and start it out as hidden.